.eslintrc

```diff
@@ -5,52 +5,41 @@
     "mocha": true,                   // mocha keywords
     "es6": true
   },
+  "plugins": [
+    "prettier"
+  ],
+  "extends": [
+    "prettier"
+  ],
   "rules": {
```

Comment:
@mishuagopian what do you think about the rules you have set to be ignored like http://eslint.org/docs/rules/one-var?